checks/automatic_dependency_update.go

```diff
@@ -20,32 +20,38 @@ import (
 	"github.com/ossf/scorecard/checker"
 )
 
-const CheckAutomaticDependencyUpdate = "Automatic-Dependency-Update"
+const checkAutomaticDependencyUpdate = "Automatic-Dependency-Update"
```

Comment:
Why are choosing to stop exporting these check names?

Reply:
these variables are not exported and not used outside this file. They are passed as arguments to `registerCheck` only, so I've renamed them.

Reply:
We do use these check names outside:

https://github.com/ossf/scorecard/blob/main/cron/worker/main.go#L183

Reply:
thanks. Will revert this hen.